docs/CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 <!-- unreleased -->
 
 ## [dev]
+- `L2Suggested` mode is now called `SuggestedPrice`
+- Removed `Optimism2` as a supported gas estimator mode
```

Comment:
Since your change is the first one in here, think we'll need the headers too.

```suggestion

### Changed

- `L2Suggested` mode is now called `SuggestedPrice`

### Removed

- Removed `Optimism2` as a supported gas estimator mode
```